psutil/__init__.py

```diff
@@ -1545,8 +1544,8 @@ def cpu_times_percent(interval=None, percpu=False):
     def calculate(t1, t2):
         nums = []
         all_delta = sum(t2) - sum(t1)
-        for field in t1._fields:
-            field_delta = getattr(t2, field) - getattr(t1, field)
+        for field in t2._fields:
+            field_delta = getattr(t2, field) - getattr(t1, field, 0)
```

Comment:
I am afraid of this change. I think it makes more sense to set `_last_cpu_times` and `_last_per_cpu_times` to None in case they cannot be determined at import time, and take that into account in this function. I just did that in e40efe13102390aa769b0de65877da57c0b3429d. Please update your branch.